uportal-war/src/main/java/org/jasig/portal/security/xslt/XalanMessageHelperBean.java

```diff
@@ -18,13 +18,14 @@
  */
 package org.jasig.portal.security.xslt;
 
-import java.util.Locale;
-
+import org.apache.commons.lang3.StringEscapeUtils;
 import org.jasig.portal.i18n.LocaleManager;
 import org.springframework.context.MessageSource;
 import org.springframework.context.MessageSourceAware;
 import org.springframework.stereotype.Service;
 
+import java.util.Locale;
```

Comment:
yes, sorry sometimes I use IntelliJ Idea... (mainly when developping javascript...)

If you want I rewrite the PR removing this re-order ?